app/app/Session.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App;
+
+use App\Enums\LocationEnum;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Session extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'start_date',
+        'end_date',
+        'location',
+        'id',
+    ];
+
+    protected $casts = [
+        'location' => LocationEnum::class,
+    ];
+
+    public function getDisplayTitleAttribute()
+    {
+        return "Session from {$this->start_date} to {$this->end_date}";
```

Comment:
Das am besten in die Translation-Datei auslagern :)

Reply:
Done